### PR TITLE
Rm joe; fix broken MD links

### PR DESCRIPTION
**Pull request by huayunh**
Fixes PXBLUE-2420

## Changes proposed in this Pull Request
- Remove Joe from current maintainers
- Fix broken markdown links

## Screenshots

image

### CODE REVIEW

File: src/__configuration__/contributors/contributors.ts [hunk 1]
```diff
@@ -106,4 +107,15 @@ export const contributors: Contributor[] = [
         description: 'Icons',
         image: RenatoMartins,
     },
+
+    {
+        name: 'Joseph Boyle',
+        description: `Front-End Developer & Strategy`,
+        image: JosephBoyle,
+        info: `Forsaking all others, Joe has been with PX Blue since its inception. A double-threat with a background in computer science and user experience, he's not afraid to drop 100+ comments on your most recent PR. Befriend at your own risk.`,
```
Comment (emclaug2) [inline src/__configuration__/contributors/contributors.ts:115]: How are the fields `info` & `contacts` used here?

**Comment by emclaug2** [comment]
Looks good just see one comment.

File: src/__configuration__/contributors/contributors.ts (hunk 1 above)
Comment (huayunh) [reply src/__configuration__/contributors/contributors.ts:115]: Can go either way; they share the same type, but info and contacts don't get rendered

Comment (emclaug2) [reply src/__configuration__/contributors/contributors.ts:115]: I'm thinking these should be two different types if those other fields aren't used.

**Review by emclaug2 — APPROVED** [review]

**Pull request merged**